docs/assemblies/custom_resources.adoc

```diff
@@ -15397,7 +15397,7 @@ AutoscalingSection defines the desired state of the autoscaling service
 
 | enabled
 | Enabled - Whether OpenStack autoscaling service should be deployed and managed
-| bool
+| *bool
```

Comment:
These changes to the docs appear completely unrelated to the PR.

Reply:
make docs wasn't added into pre-commit hook, I added it here: https://github.com/openstack-k8s-operators/openstack-operator/pull/851